src/main/js/src/containers/OldPaymentRedirect/OldPaymentRedirect.js

```diff
@@ -1,80 +0,0 @@
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
-import { withTranslation } from 'react-i18next';
-
-import axios from '../../axios';
-import Alert from '../../components/Alert/Alert';
-import classes from './OldPaymentRedirect.module.css';
-
-export class OldPaymentRedirect extends Component {
-  paymentForm = React.createRef();
-
-  state = {
-    formData: null,
-    error: false,
-  };
-
-  componentDidMount = () => {
-    const {
-      match: { params },
-    } = this.props;
-    axios
-      .get(`/yki/payment/formdata?registration-id=${params.registrationId}`)
-      .then(({ data }) => {
-        this.setState({ formData: data });
-      })
-      .catch(err => {
-        this.setState({ error: true });
-      });
-  };
-
-  componentDidUpdate = () => {
-    if (this.state.formData) {
-      this.paymentForm.current.submit();
-    }
-  };
-
-  render() {
-    return this.state.formData && !this.state.error ? (
-      <form
-        ref={this.paymentForm}
-        action={this.state.formData.uri}
-        method="POST"
-        acceptCharset="ISO-8859-1"
-      >
-        {this.state.formData.params.PARAMS_IN.split(',').map((p, i) => {
-          return (
-            <input
-              key={i}
-              name={p}
-              type="hidden"
-              value={this.state.formData.params[p]}
-            />
-          );
-        })}
-        <input
-          name="AUTHCODE"
-          type="hidden"
-          value={this.state.formData.params.AUTHCODE}
-        />
-      </form>
-    ) : (
-      this.state.error && (
-        <>
-          <main id="main" className={classes.Content}>
-            <Alert
-              title={this.props.t('payment.redirect.error')}
-              optionalText={this.props.t('payment.redirect.error.info')}
```

Comment:
Nämä lokalisaatiot voinee myös poistaa (jos ei uuden flown käytössä)

Reply:
On käytössä.

Reply:
On edelleen käytössä.

Reply:
Joo, mulla tulee lähinnä mieleen että kun varsinaista uutta maksuintegraatiotakaan ei tunnut kunnolla katselmoitua eikä sitä myöten perehdyttyä edes tähän alkuperäiseen toteutukseen, että mergaisko myös tämän ja backendin PR:n ilman kunnon katselmointia. Jos lähinnä testais että kaikki toimii edelleen kuten pitää.

Onko teille @kimmotaskinen ja @jrkkp mielipidettä tästä että katsoisko näitä PR:iä tarkemminkin läpi vai vaan silmäillen, testaa ja vie käyttöön?

Reply:
Juu, kyllähän tää sinänsä toiminee edelleen, ei tässä oikein muuta tehty kuin poistettu käytöstä poistunutta koodia. Bäkkärin puolella ehkä vähän enemmän tekemistä, mutta tän pitäis olla aika suoraviivainen. Pitää tietty vielä testiympäristöön viedä.

Reply:
Joo, bäkkäripuolta enemmän mietinkin että onhan siellä varmasti kaikki yhä ok. Mutta mun puolesta ainakin riittäis ihan vaan testiympäristössä pyörittely tuon varmistamiseen